build.gradle

```diff
@@ -1,119 +1,35 @@
-    id 'com.github.ben-manes.versions' version '0.28.0'
-    id 'org.owasp.dependencycheck' version '5.3.2.1'
-    id 'se.patrikerdes.use-latest-versions' version '0.2.13'
-}
-
-repositories {
-    mavenLocal()
-    mavenCentral()
-}
-
-def isNonStable = { String version ->
-    def stableKeyword = ['RELEASE', 'FINAL', 'GA'].any { it -> version.toUpperCase().contains(it) }
-    def regex = /^[0-9,.v-]+(-r)?$/
-    return !stableKeyword && !(version ==~ regex)
-}
-
-dependencyUpdates {
-    rejectVersionIf {
-        isNonStable(it.candidate.version) && !isNonStable(it.currentVersion)
-    }
-}
-
-configurations {
-    agentDeps
-    agentDeps.transitive = false
 }
 
-dependencyCheck {
-  failBuildOnCVSS = 5
-  suppressionFiles = ['dependency-check-suppress-spring.xml']
-  format = 'ALL'
-  analyzers {
-    assemblyEnabled = false
-  }
-}
-project.sourceCompatibility = 11
-project.targetCompatibility = 11
-
 group = 'uk.gov.justice.digital.delius'
 
-def today = Instant.now()
-def todaysDate = LocalDate.now().format('yyyy-MM-dd')
-version = System.getenv('CI') ? "${todaysDate}.${System.getenv('CIRCLE_BUILD_NUM')}" : todaysDate
-
-bootJar {
-    manifest {
-        attributes 'Implementation-Title': rootProject.name, 'Implementation-Version': rootProject.version
-    }
-}
-
-springBoot {
-    buildInfo {
-        properties {
-            artifact = rootProject.name
-            version =  version
-            group = 'uk.gov.justice.digital.hmpps'
-            name = rootProject.name
-            time = today
-            additional = [
-                    by                   : System.properties['user.name'],
-                    operatingSystem      : "${System.properties['os.name']} (${System.properties['os.version']})",
-                    continuousIntegration: System.getenv('CI') ? true : false,
-                    machine              : InetAddress.localHost.hostName
-            ]
-        }
-    }
-}
-
-dependencyManagement {
-    imports {
-        mavenBom(org.springframework.boot.gradle.plugin.SpringBootPlugin.BOM_COORDINATES)
-    }
-}
+ext["spring-security.version"] = "5.3.2.RELEASE"
 
 dependencies {
     implementation files('lib/ojdbc8-12.2.0.1.jar')
 
     implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.11.0'
     implementation 'io.jsonwebtoken:jjwt:0.9.1'
-    implementation 'org.springframework.boot:spring-boot-starter-web'
     implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
     implementation 'org.springframework.boot:spring-boot-starter-data-ldap'
-    implementation 'org.springframework.boot:spring-boot-starter-actuator'
     implementation 'org.springframework.boot:spring-boot-starter-security'
     implementation 'org.springframework.security.oauth:spring-security-oauth2:2.4.1.RELEASE'
     implementation 'org.springframework.security:spring-security-jwt:1.1.0.RELEASE'
 
     implementation 'org.hibernate:hibernate-java8:5.4.5.Final'
```

Comment:
really old version of hibernate here - line can be removed as `data-jpa` will grab it instead.  Should be `hibernate-core` and then it will be updated to 5.4.15, but best removing.